recipes/glaze/all/conanfile.py

```diff
@@ -19,44 +19,26 @@ class GlazeConan(ConanFile):
     package_type = "header-library"
     settings = "os", "arch", "compiler", "build_type"
     no_copy_source = True
-
-    @property
-    def _min_cppstd(self):
-        return "20" if Version(self.version) < "3.0.0" else "23"
-
-    @property
-    def _compilers_minimum_version(self):
-        return {
-            "20": {
-                "Visual Studio": "17",
-                "msvc": "193",
-                "gcc": "11" if Version(self.version) < "2.6.3" else "12",
-                "clang": "14",
-                "apple-clang": "13.1",
-            },
-            "23": {
-                "Visual Studio": "17",
-                "msvc": "193",
-                "gcc": "12",
-                "clang": "15",
-                "apple-clang": "14",
-            },
-        }.get(self._min_cppstd, {})
+    options = {
+        "with_ssl": [True, False],
+    }
+    default_options = {
+        "with_ssl": False,
+    }
 
     def layout(self):
         basic_layout(self, src_folder="src")
 
     def package_id(self):
         self.info.clear()
 
+    def requirements(self):
+        self.requires("asio/1.34.2", transitive_headers=True)
+        if self.options.with_ssl:
+            self.requires("openssl/[>=1.1 <4]")
+
```

Comment:
```suggestion
```

I would ask you to remove dependencies: They are very optional, for instance, asio is only used as a header in the extension folder. Doing a quick search, most of package managers also don't add those dependencies: https://repology.org/project/glaze/versions

The readme also doesn't mention these options, you need to dig into docs folder to find them too: https://github.com/stephenberry/glaze/blob/v5.7.2/README.md

It's not really clear how useful it would be in CCI due to the complexity added by those dependencies, but no usage by users, so I would prefer waiting for an issue to appear, asking about and detailing the case.